backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/utils/DeviceUtil.kt

```diff
@@ -19,24 +19,32 @@
 package net.skyscanner.backpack.compose.utils
 
 import androidx.compose.runtime.Composable
-import androidx.compose.ui.platform.LocalConfiguration
+import androidx.compose.ui.platform.LocalDensity
+import androidx.compose.ui.platform.LocalWindowInfo
+import androidx.compose.ui.unit.dp
 
 @Composable
 internal fun isSmallTablet(): Boolean {
-    val configuration = LocalConfiguration.current
-    return configuration.screenWidthDp > SMALL_TABLET_WIDTH
+    val widthDp = with(LocalDensity.current) {
+        LocalWindowInfo.current.containerSize.width.toDp()
+    }
+    return widthDp > SMALL_TABLET_WIDTH.dp
 }
 
 @Composable
 internal fun isTablet(): Boolean {
-    val configuration = LocalConfiguration.current
-    return configuration.screenWidthDp > TABLET_WIDTH
+    val widthDp = with(LocalDensity.current) {
+        LocalWindowInfo.current.containerSize.width.toDp()
+    }
+    return widthDp > TABLET_WIDTH.dp
 }
 
 @Composable
 internal fun isDesktop(): Boolean {
-    val configuration = LocalConfiguration.current
-    return configuration.screenWidthDp > TABLET_WIDTH
+    val widthDp = with(LocalDensity.current) {
+        LocalWindowInfo.current.containerSize.width.toDp()
+    }
+    return widthDp > TABLET_WIDTH.dp
```

Comment:
I think this is meant to be
```suggestion
    return widthDp > DESKTOP_WIDTH.dp
```

Reply:
true! nice catch, luckily this is still a draft PR